exporter/f5cloudexporter/config.go

```diff
@@ -0,0 +1,73 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package f5cloudexporter
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+
+	otlphttp "go.opentelemetry.io/collector/exporter/otlphttpexporter"
+)
+
+// Config defines configuration for F5 Cloud exporter.
+type Config struct {
+	// Config represents the OTLP HTTP Exporter configuration.
+	otlphttp.Config `mapstructure:",squash"`
```

Comment:
Is all the config from `otlphttp.Config` supported? If not we should just add the fields directly that are.

Reply:
Yes, all of the fields are supported at this moment; however, in the future when we move this out of beta will likely reduce them to the minimum that is supported.